tests/testutil/service.go

```diff
@@ -284,7 +284,7 @@ func (ts *TestService) GetAppEndpointResponse() string {
 
 // Returns count of ready and available pods
 func (ts *TestService) GetAvailableReplicas() int {
-	if ts.Service.Status.DeploymentReady && ts.Service.Status.ScaleStatus != nil {
+	if ts.Service.Status.DeploymentReady && ts.Service.Status.ScaleStatus != nil || (&ts.Service.Status.DeploymentReady == nil && ts.Service.Status.ScaleStatus != nil) {
```

Comment:
Why are you checking the address of `DeploymentReady` ? Can't you just check if true or false.

Reply:
essentially it has 3 states in the manifest: `true`, `false`, or nonexistent (nil address). I want to get the scale status if it is true or nonexistent, because during scale up/down there can be available replicas and the service will be ready, but this field will not be present.